EMOBON00092/provenance.json

```diff
@@ -0,0 +1,64 @@
+{ "@context": {
+    "@base":"https://github.com/emo-bon/",
+    "@vocab":"https://w3id.org/ro/crate/1.2-DRAFT/context"
+},
```

Comment:
is the @id for refering to the CPMProvenanceFile  at ln 8 below included in this context? 
if not should be added imho